### ISSUE TITLE
Error: json.Unmarshal - invalid character '<' looking for beginning of value

### ISSUE THREAD

**Issue by tofkamp** [terraform-routeros/terraform-provider-routeros#275]
I was working with ansible to try DSC, but a friend told me about terraform. So I am trying terraform with routeros 7.11.2
Created an own CA, and webserver. Added the CA to trusted store of ubuntu. Set this webserver certificate  on www-ssl. Just trying to get a working example, by setting the /system/identity. I included the file, by adding ".txt" to the filename  so github accepts it.
To reproduce the error : Put file in empty folder, terraform init; terraform plan; terraform apply
```
routeros_system_identity.identity: Creating...
routeros_interface_bridge.bridge: Creating...
╷
│ Error: json.Unmarshal - invalid character '<' looking for beginning of value
│
│   with routeros_interface_bridge.bridge,
│   on main.tf line 23, in resource "routeros_interface_bridge" "bridge":
│   23: resource "routeros_interface_bridge" "bridge" {
│
╵
╷
│ Error: json.Unmarshal - invalid character '<' looking for beginning of value
│
│   with routeros_system_identity.identity,
│   on main.tf line 29, in resource "routeros_system_identity" "identity":
│   29: resource "routeros_system_identity" "identity" {

```
It complains about invalid character "<", which is not in the file, nor in the examples on website.
I have installed terraform-routeros 1.18.2, on terraform 1.6.0. I thought it was an error in my system, but I have done the docker tutorial, and all went well. No errors. 
[main.tf.txt](https://github.com/terraform-routeros/terraform-provider-routeros/files/12837943/main.tf.txt)
What is going wrong ? Is the format changed ? Is there a program error ?

**Comment by vaerh** [comment]
Hi, I haven't gotten to version 7.11 yet and maybe it is the version that is causing the problem. But it should be checked at the lab.
Can I ask you to run ```ROS_LOG_COLOR=1 TF_LOG=debug terraform apply``` and post the output?

**Comment by tofkamp** [comment]
[err.txt](https://github.com/terraform-routeros/terraform-provider-routeros/files/12840175/err.txt)
In the output I saw the following:
error response body:
<!doctype html>
<html lang=en>
<title>Error 501 : Not Implemented</title>
<h1>Error 501 : Not Implemented</h1>

Just to be sure it can work, I did: curl -k -u admin:admin https://bh1-k1-sw1.home.arpa/rest/system/resource
I got some json response with sensible numbers.

**Comment by tofkamp** [comment]
I found the cause of this error. It was the trailing "/" of the hosturl.
In the trace I noticed the dubble "//" after the hostname. Normal it is no problem, and other webservers just skips the extra "/".
For an user it is normal to put a "/" after the hostname. In order to prevent errors, or abandoning adoption of this provider, put a message in the manual, or do a check in the program, or correct it automatically.
